exotations/exotica_core_task_maps/init/joint_pose.in

```diff
@@ -2,5 +2,5 @@ class JointPose
 
 extend <exotica_core/task_map>
 
-Optional Eigen::VectorXd JointRef;
-Optional Eigen::VectorXd JointMap;
+Required Eigen::VectorXd JointRef;
```

Comment:
These should remain `Optional` with a default argument: `Optional Eigen::VectorXd JointRef = Eigen::VectorXd();`